code/game/atoms_movable.dm

```diff
@@ -88,6 +88,9 @@
 			if(loc == newloc) //Remove this check and people can accelerate. Not opening that can of worms just yet.
 				newtonian_move(last_move)
 
+	for(var/mob/M in contents)
+		M.Move(newloc,direct)

```

Comment:
Did I forget to comment on this? mob/Move() being called while they are inside containers is, to say the least, completely fucking insane. Clowns making boot noises while in lockers or flying through space in a spacepod.

Snowflaking 
```dm
for(var/mob/M in contents)
    if(M.hud_used)
        M.hud_used.update_parallax()
```
is actually better.